src/map/utils/charutils.cpp

```diff
@@ -2302,6 +2302,89 @@ namespace charutils
         }
     }
 
+    void SaveJobChangeGear(CCharEntity* PChar)
+    {
+        if (PChar == nullptr)
+        {
+            return;
+        }
+
+        const char* Query  = "REPLACE INTO char_equip_saved VALUES (%u, %u, %u, %u, %u, %u, %u, %u, %u, %u, %u, %u, %u, %u, %u, %u, %u, %u);";
```

Comment:
In the spirit of other queries, we may want to explicitly assign these values when inserting